config/README.md

```diff
@@ -0,0 +1,50 @@
+ * **root** {string} : default `'dist/'` - Root directory for `webpack-dev-server`. All static assets you need for your prototype(s) should be located in this directory. The `output` option should be relative to this path.
+ * **sectionTemplate** {string} : default `path.join(__dirname, '../templates/section.hbs')` - Override for the template used to produce styleguide sections. This should always be a handlebars file, and will utilize the KSS data as its source.
+ * **templates** {object} : default
+ 	```js
+ 	{
+      loader: {
+        test: /\.hbs$/,
+        loader: 'handlebars-loader',
+      },
+      extension: '.hbs',
+    }
+    ```
+    Object containing a webpack loader for your template files and a corresponding file extension. Instructions for configuring a webpack loader can be found [here](https://webpack.github.io/docs/configuration.html#module-loaders).
+ * **window** {object} : default `{}` - Object containing variables that should be attached to the global `window` object on every prototype.
+
+## Webpack
+Huron will attempt to merge its own configuration requirements for webpack with your local configuration, allowing you to use the same build tools (and resulting scripts and styles) in your prototypes as you do on your live site.
+ * Huron will automatically add `webpack-dev-server/client?http://localhost:${huron.port}`, `webpack/hot/dev-server` and the Huron frontend script to your configured entry point, so you do not need to worry about configuring this. In addition, Huron will de-dupe the `HotModuleReplacementPlugin()` if you're using it in your production webpack configuration.
+ * Huron will automatically add configurations for the `html-loader` and `json-loader` for HTML templates and JSON data respectively
+ * The HTML webpack plugin will be largely configured automatically, so you're only required to supply the title of the prototype. As described above, however, you may reconfigure the options. Just beware that this may break your setup, especially if you override the `chunks` option. NOTE: this is also the method by which you may supply a custom ejs template for your prototypes. Again, do this at your own risk and consult the [default template](../templates/prototype-teplate.ejs) first.
+ * Huron will override the `devServer` property in your webpack config in favor of its own.
```

Comment:
Might be worth noting that you cannot override the devServer.stats object in a local config